### ISSUE TITLE
[Adaptive Cards - Designer] - Ensures ARIA attributes are not prohibited for an element's role (.acd-peer[title="Double click to edit"][aria-label="TextBlock"]:nth-child(2))

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9033]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/samples/ProductVideo.html
OS Version: 22H2(OS Build 25300.1000)
Browser Version: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on the Edge Dev browser.
2. Press tab key and navigate to the designer tab and press enter key to activate it.
3. Run the tool and observe the issue.

**Actual Result:**
ARIA attributes are prohibited for an element's role (.acd-peer[title="Double click to edit"][aria-label="TextBlock"]:nth-child(2))

**Expected Result:**
Ensures ARIA attributes are not prohibited for an element's role (.acd-peer[title="Double click to edit"][aria-label="TextBlock"]:nth-child(2))

Element path: .acd-peer[title="Double click to edit"][aria-label="TextBlock"]:nth-child(2)

Snippet: <div class="acd-peer" tabindex="0" title="Double click to edit" aria-label="TextBlock" style="position: absolute; z-index: 10; display: initial; width: 378.4px; height: 22.6125px; left: 10px; top: 10px;"></div>

How to fix: 
Fix all of the following:
  aria-label attribute cannot be used on a div with no valid role attribute.

**User Impact:**
It will impact the AT user if the ARIA attributes are prohibited for any controls on the screen.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Ensures ARIA attributes are not prohibited for an element's roles](https://github.com/user-attachments/assets/7a9c2df2-ff82-44b0-b2f8-1a5530081cda)

![Ensures ARIA attributes are not prohibited for an element's role](https://github.com/user-attachments/assets/166e88d7-fca2-41a4-a3aa-f18e8b85a98a)

### Card JSON

```JSON
NA
```
